File: Frontend/Private/gco-private/src/app/model/core/all-around-scoreboard.ts
import { Apparatus } from './apparatus'
import { AllAroundScoreboardSlot} from './all-around-scoreboard-slot'

export interface AllAroundScoreboard{
    id: string;
    competitionId: string;
    ageCategory: string;
    tieBrake: boolean;
    apparatuses: Apparatus[];
    slots: AllAroundScoreboardSlot[];
}

File: Backend/scoring_service/core/domain/all_around_scoreboard.go
package domain

import (
	"github.com/google/uuid"
)

type AllAroundScoreboard struct {
	ID            uuid.UUID
	CompetitionID uuid.UUID
	Competition   Competition
	AgeCategory   string
	TieBrake      bool
	Apparatuses   []Apparatus `gorm:"serializer:json"`
	Slots         []AllAroundScoreboardSlot
}
